DevelopGitHub/GitHubExtension/GitHubExtension.Activity.External.WebAPI/Extensions/CreateHttpMessageExtension.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using GitHubExtension.Activity.External.WebAPI.Queries;
+using GitHubExtension.Constant;
+
+namespace GitHubExtension.Activity.External.WebAPI.Extensions
+{
+    public static class CreateHttpMessageExtension
+    {
+        private static readonly Dictionary<string, string> DefaultHeaders = new Dictionary<string, string>()
+        {
+             //Need to set user-agent to access GitHub API, Using Chrome 48
+            { GitHubConstants.UserAgentHeader, GitHubConstants.UserAgentHeaderValue }
+        };
+
+        public static HttpRequestMessage CreateHttpMessage(this IGitHubEventsQuery eventsQuery, HttpMethod method, string requestUri)
```

Comment:
Can you explain naming of that class is you are extend IGitHubEventsQuery not HttpRequestMessage?